HYPER_RECIPES.md

```diff
@@ -6,12 +6,10 @@ Even though you could try and convince us to hire you for your good humor, charm
 
 In Hyper Recipes, you'll build an application that will present a collection of recipes fetched from our [custom backend endpoint](http://hyper-recipes.herokuapp.com/recipes). Other endpoints are available if you are feeling adventurous. You decide the level of complexity, functionality and the look and feel of the application. We appreciate good looking apps and thoughtful user experiences, Google Play Store is a great source of inspiration.
 
-We tend to enjoy some popcorn while reviewing your app, reading the code as if it was a fantasy book filled with dragons and unicorns. Please make it shine.
+If you want to have go at the recipes app (or we urge you to), we'll make you a nice and clean repo which you will fill with marvellous code. In our daily work, we work by submitting pull requests for all our features, updates, refactorings and bugfixes, and that's how we would like you to work, too. So start out by cloning the repo, create a new branch and start committing. Remember to keep the commits as small and specific as possible. When you have a feature done (or you want our comment on something), push it to github and create a pull request, so we can review and comment on it. We tend to enjoy some popcorn while reviewing your app, reading the code as if it was a fantasy book filled with dragons and unicorns. Please make it shine.
```

Comment:
Well, wouldn't you start by running `git clone reponame`? But maybe "start out by creating a new branch" is better anyway? Or how would you formulate that sentence?